doc/help.md

```diff
@@ -1,70 +1,205 @@
+
+If you are a member of a publisher, you can use the **My pub.dev > My
+publishers** to view a list of those publishers, and under those the packages
+published with those.
+
+## Discontinuing a package
+
+Keep in mind that publishing is forever. As soon as you publish your package,
+users can depend on it. Once they start doing that, removing the package would
+break theirs. To avoid that, the [pub.dev policy](https://pub.dev/policy)
+disallows unpublishing packages except for very few cases.
+
+If you are no longer maintaining a package, you can [mark it
+discontinued](https://dart.dev/tools/pub/publishing#discontinue), and it will
+disappear from pub.dev search.
+
+# Package scores & pub points
+
+*NOTE: The Pub scoring model evolves over time, and is likely to be extended
+with additional checks in the future.*
+
+For each package, this site displays three scoring dimensions. These are
+displayed in search results, in the sidebar on individual package pages, and in
+full detail in the scoring report on the 'Scores' tag of an individiual package.
+The three dimensions are:
+
+  - **Likes**: A measure of how many developers have liked a package. This
+    provides a raw measure of the overall sentiment of a package from peer
+    developers.
+  - **Pub Points**: A new measure of quality. This includes several dimensions
+    of quality such as code style, platform support, and maintainability. More
+    about this below.
+  - **Popularity**: A measure of how many developers use a package, providing
+    insight into what other developers are using.
+
+## Likes
+
+Likes offer a measure of how many developers have liked a package. To like a
+package, locate the **thumbs up button** located in the upper-right corner of a
+package page.
+
+To view packages you have liked, use the **My pub.dev > My liked packages** menu
+option.
+
+## Popularity
+
+Popularity measures the number of apps that depend on a package over the past 60
+days. We show this on a normalized scale from 100% (the most used package) to 0%
+(the least used package), but are investigating if we can provide absolute usage
+counts in a future version.
+
+Although this score is based on actual download counts, it compensates for
+automated tools such as continuous builds that fetch the package on each change
+request.
+
+## Pub Points
+
+Pub points is pub.dev's measure of quality. Pub points are awarded in five categories:
+
+### Follow Dart file conventions
+
+Dart packages are expected to follow certain file conventions for how to
+[organize a package](https://dart.dev/guides/libraries/create-library-packages#organizing-a-library-package.
+Most importantly make sure to:
+
+  - Provide a [`pubspec.yaml` file](https://dart.dev/tools/pub/pubspec). Ensure
+    all Urls are valid and use a secure `https:` scheme.
+
+  - Provide a [](https://dart.dev/tools/pub/package-layout#license) file using
+    an an [OSI-approved license](https://opensource.org/licenses).
```

Comment:
We do not require OSI-approved licenses. I think encouraging it would be great, but afaik we have no scoring on this.

Reply:
Only scoring on the fact that a license is present.

Reply:
Added preferably